### PR TITLE
fix: temp disable transfer on Base

### LINKED ISSUE #10402: Temp fix: Disable transfer on Base
### Is there an existing issue?

- [X] I have searched the existing issues

### What happened?

As @prury correctly find out the transfer won't work on Base

- #10397

### Please reproduce in steps

1. Go to transfer
2. send amount
3. Nothing will happen

### Expected Behavior

I expect user that he/she won't get to transfer / teleport from kodadot/base

### What browsers are you seeing the problem on?

_No response_

### At which address did you encounter bug?

_No response_

### Are you logged in?

None

### Which wallet you are using?

_No response_

### At which chain did you encounter bug?

_No response_

### Screenshots

-

### Wallet Address & Transaction hash

_No response_

### Relevant log output

_No response_

### Payment link for reward

https://canary.kodadot.xyz/dot/transfer/?target=<My_Polkadot_Address_check_https://github.com/kodadot/nft-gallery/blob/main/REWARDS.md#creating-your-dot-address>

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [Koda - Generative Art Marketplace](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix

## Needs QA check

- @kodadot/qa-guild please review

## Context

- [x] Closes #10402

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Review by vikiival — CHANGES_REQUESTED** [review]
this is very bad :D

User won't have idea that he/she is suddenly on AHP.

Intended behaviour is not to show them transfer

File: components/common/ConnectWallet/WalletAssetMenu.vue [hunk 1]
```diff
@@ -61,27 +61,29 @@ import { NeoDropdown, NeoDropdownItem, NeoIcon } from '@kodadot1/brick'
...
+    {
+      label: 'Onchain Identity',
+      to: '/identity',
+    },
+    {
+      label: 'Migrate',
+      to: '/migrate',
+    },
+  ].filter((menu) => !isBase.value || menu.label !== 'Transfer'),
+)
```
Comment (vikiival) [inline components/common/ConnectWallet/WalletAssetMenu.vue:86]: None of this is available on base :) also think it was solved by @hassnian

**Review by vikiival — CHANGES_REQUESTED** [review]

File: components/common/ConnectWallet/WalletAssetMenu.vue (hunk 1 above)
Comment (hassnian) [reply components/common/ConnectWallet/WalletAssetMenu.vue:86]: https://github.com/kodadot/nft-gallery/blob/578f43b1ae37c9338e6eaf9daea90684de40b44d/components/common/ConnectWallet/WalletAssetMenu.vue#L70

Comment (Jarsen136) [reply components/common/ConnectWallet/WalletAssetMenu.vue:86]: Then I guess we could close this issue as it has been implemented on other PR.

**Comment by Jarsen136** [comment]
REF: https://github.com/kodadot/nft-gallery/pull/10405#discussion_r1630779730

**Pull request closed without merging**